api/utils/addr.go

```diff
@@ -44,15 +44,15 @@ func ParseAddr(addr string) (string, error) {
 	case "unix":
 		return u.Path, nil
 	case "http", "https":
-		fmt.Println(u.Host)
 		return u.Host, nil
 	default:
 		return "", trace.BadParameter("'%v': unsupported scheme: '%v'", addr, u.Scheme)
 	}
 }
 
 // ParseHost strings like "tcp://host:port/path" and returns "host".
-func ParseHost(addr string) (string, error) {
+func ParseHost(addr string) (ra string, err error) {
+	defer fmt.Printf("\naddr: %v, return: %v, err: %v\n\n", addr, ra, err)
```

Comment:
Just a debugging leftover?
```suggestion
func ParseHost(addr string) (string, error) {
```